arm/Microsoft.Network/connections/deploy.bicep

```diff
@@ -1,8 +1,8 @@
 @description('Required. Remote connection name')
 param connectionName string
 
-@description('Required. Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways')
-param vpnSharedKey string
+@description('Specifies a VPN shared key. The same value has to be specified on both Virtual Network Gateways')
```

Comment:
If its optional now, add `Optional.` in front on description. Also run the script to refresh the readme with the corresponding update there.

Reply:
In the wiki we have some guidelines we want to follow. Visit [Bicep Module Guidelines](https://github.com/Azure/ResourceModules/wiki/ModulesDesign#bicep-template-guidelines) in our Wiki for more info

Reply:
Updated the param and reviewed the guidelines 😀 thanks

Reply:
Please also update the readme. For ease, there is a script here to help you.
Try the following:
``` powershell
. '.github\workflows\scripts\Set-ModuleReadMe.ps1'
Set-ModuleReadMe -TemplateFilePath arm/Microsoft.Network/connections/deploy.bicep
```